.github/workflows/test-nix.yml

```
@@ -0,0 +1,36 @@
+---
+name: Test Nix package and environment
+
+on:
+  schedule:
+    - cron: '0 1 * * 1'
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  test-nix:
+    runs-on: ubuntu-22.04
+
```

Select the permissions needed. Does it use attestations too? (https://docs.github.com/en/actions/security-guides/using-artifact-attestations-to-establish-provenance-for-builds)
```suggestion
  permissions:
    contents: read
```
https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

---

But cloning a public repo shouldn't need that permission though.

---

> Does it use attestations too ?

Not sure, will see :).